packages/manager/src/featureFlags.ts

```diff
@@ -129,6 +129,7 @@ export interface Flags {
   aclpLogs: BetaFeatureFlag;
   aclpReadEndpoint: string;
   aclpResourceTypeMap: CloudPulseResourceTypeMapFlag[];
+  aclpServices: AclpServices;
```

Comment:
```suggestion
  aclpServices: Partial<AclpServices>;
```
and also remove aclpBetaServicesFlag